helm/reana-dev/templates/NOTES.txt

```diff
@@ -0,0 +1 @@
+Thanks for flying REANA
```

Comment:
... with a rocket emoticon?!

Reply:
yesss! I was a bit sad when removing it 😃